book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_reader :rental
+  attr_accessor :author, :title
+
+  def initialize(author, title)
+    @author = author
+    @title = title
+    @rental = []
+  end
```

Comment:
- [ ] as part of the requirements a `method for adding rentals should be present` , kindly add the method like the example below and make sure to update the rental to be `attr_accessor` instead of `attr_reader`

```ruby
def add_rental(person, date)
    Rental.new(date, self, person)
  end
```